packages/ui/src/components/Select/select.tsx

```diff
@@ -9,28 +10,54 @@ import type { TypographyTextProps } from '@open-condo/ui/src'
 import { sendAnalyticsChangeEvent, extractChildrenContent } from '@open-condo/ui/src/components/_utils/analytics'
 
 import { SELECT_TYPES } from './constants'
+import { useItems } from './hooks/useItems'
+
+import { Either } from '../_utils/types'
 
 import type { SelectProps as DefaultSelectProps } from 'antd'
 
-const SELECT_CLASS_PREFIX = 'condo-select'
-const Option: typeof DefaultSelect.Option = DefaultSelect.Option
-const OptGroup: typeof DefaultSelect.OptGroup = DefaultSelect.OptGroup
+export const SELECT_CLASS_PREFIX = 'condo-select'
+export const Option: typeof DefaultSelect.Option = DefaultSelect.Option
```

Comment:
Why you specifying type by yourself?
```
const a = 5 // typeof a = typeof 5
```

Reply:
It's already been written, I will remove the excess :)

Reply:
Got it 👍